api/envoy/api/v2/core/protocol.proto

```diff
@@ -142,6 +142,10 @@ message Http1ProtocolOptions {
   //   - Not a response to a HEAD request.
   //   - The content length header is not present.
   bool enable_trailers = 5;
+
+  // [#not-implemented-hide:]
+  // Overrides envoy's behavior on invalid HTTP messaging configured in the hcm
```

Comment:
you'll want to ref link to the actual hcm field - you can crib that notation from my PR as well :-)